docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst

```diff
@@ -128,19 +158,339 @@ to create the rescue image. If you used the configuration example above, you wil
    In this configuration it will only create the rescue ISO
    (same as the :command:`rear mkrescue` command).
 
+.. note::
+
+   The rescue image is created from the running system.
```

Comment:
Maybe add the info that it also contains the Bareos client and related configuration to perform the restore?

Reply:
added.